bitseq/store.go

```diff
@@ -70,6 +70,12 @@ func (h *Handle) Exists() bool {
 	return h.dbExists
 }
 
+// DataScope method returns the storage scope of the datastore
+func (h *Handle) DataScope() datastore.DataScope {
+	//This method is useless currently, hence return a arbitrary value here
+	return datastore.DataScope(-1)
```

Comment:
I think it should be Global as the main intention of `bitseq` is to provide a distributed guranteed unique identity out of a bitfield. ping @mrjana @aboch WDYT ?